spec/lib/twingly/http_spec.rb

```diff
@@ -468,6 +468,19 @@ class CustomError < StandardError; end
     end
   end
 
+  describe "#initialize" do
```

Comment:
I think I would "cherry-pick" a bit more, such as:

```diff
diff --git a/spec/lib/twingly/http_spec.rb b/spec/lib/twingly/http_spec.rb
index a8ac842..81136ad 100644
--- a/spec/lib/twingly/http_spec.rb
+++ b/spec/lib/twingly/http_spec.rb
@@ -469,13 +469,17 @@ RSpec.describe Twingly::HTTP::Client do
   end
 
   describe "#initialize" do
-    let(:client) { described_class.new(base_user_agent: base_user_agent) }
+    describe "default logger" do
+      subject(:logger) do
+        client = described_class.new(base_user_agent: base_user_agent)
 
-    context "when no logger is given" do
-      it { expect(client.logger).to be_a(Logger) }
+        client.logger
+      end
+
+      it { is_expected.to be_a(Logger) }
 
       it "suppresses all output" do
-        expect { client.logger.info("foo") }
+        expect { logger.info("foo") }
           .to_not output(/foo/).to_stdout
       end
     end
```

My reasoning here is that the `let` of the `client` is not that interesting for the test cases here, we are focusing on the logger

Reply:
or.. a hybrid (in progress...):

Reply:
Yeah, sure. The hybrid is now in f679e7e